cmd/gpu_plugin/gpu_plugin_test.go

```diff
@@ -696,3 +714,253 @@ func TestBypath(t *testing.T) {
 		}
 	}
 }
+
+func TestPciDeviceForCard(t *testing.T) {
+	root, err := os.MkdirTemp("", "test_pci_device_for_card")
+	if err != nil {
+		t.Fatalf("can't create temporary directory: %+v", err)
+	}
+	// dirs/files need to be removed for the next test
+	defer os.RemoveAll(root)
+
+	sysfs := path.Join(root, "sys")
+
+	cardPath := filepath.Join(sysfs, "class", "drm", "card0")
+	cardDevicePath := filepath.Join(cardPath, "device")
+
+	if err := os.MkdirAll(cardDevicePath, 0750); err != nil {
+		t.Fatalf("couldn't create card's device path")
```

Comment:
```suggestion
		t.Fatalf("card device path creation failed: %v", err)
```

Or: 
```suggestion
		t.Fatalf("failed to create device path for card: %v", err)
```

And similar changes for rest of error messages, so that their phrasing is consistent and the actual error is output.